### PR TITLE
ZIR-152: Automatically populate issues in linear form PRs

### PR DESCRIPTION
**Pull request by shkoo**

### CODE REVIEW

File: .github/workflows/linear.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  create-linear-issue-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find or create a Linear Issue
+        uses: risc0/action-find-or-create-linear-issue@risc0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          linear-api-key: ${{ secrets.LINEAR_API_KEY }}
+          linear-team-key: "ZIR"
+          linear-created-issue-state-id: "bab82211-2372-4199-89d3-71f1246cee71" # in-progress
```
Comment (jacobdweightman) [inline .github/workflows/linear.yml:28]: It seems weird to me that a state ID would look like a UUID... but as long as this means "in progress" and leaks nothing else, I guess that's fine. @mothran ?

Comment (jacobdweightman) [reply .github/workflows/linear.yml:28]: ... Wow. I just grepped through the HTML for the Linear board, and that's pretty cursed. But I guess LGTM.

**Review by jacobdweightman — COMMENTED** [review]
LGTM, but I'll defer to @mothran to approve

File: .github/workflows/linear.yml (hunk 1 above)
Comment (mothran) [reply .github/workflows/linear.yml:28]: Yeah to get that either you have to use the API / graphql client OR do what I did and change the description of the state in the UI while you have chrome debug tools monitoring the network tab to see what the UUID it references on update is.

**Review by mothran — APPROVED** [review]

**Pull request merged**